src/components/autocomplete/autocomplete.html

```diff
@@ -0,0 +1,23 @@
+<m-dropdown :filterable="true"
+            :clear-invalid-selection-on-close="true"
+            :show-arrow-icon="false"
+            :enable-animation="false"
+            :include-filterable-status-items="false"
+            :placeholder="placeholder"
+            :focus="focus"
+            :disabled="disabled"
+            :error-message="errorMessage"
+            :valid-message="validMessage"
+            :helper-message="helperMessage"
+            :width="width"
+            :max-width="maxWidth"
+            :label="label"
+            :required-marker="requiredMarker"
+            :tag-style="tagStyle"
+            v-model="selection"
+            @input="onInputChangeThrottled">
+    <m-dropdown-item v-for="i in items"
```

Comment:
toujours utiliser :key avec v-for ref: https://vuejs.org/v2/style-guide/#Keyed-v-for-essential

Reply:
ajoute